release-notes/latest.md

```diff
@@ -69,3 +69,23 @@ Currently the only API action that can be limited in this way is `ListAllJobs` -
 #### <sub><sup><a name="5624" href="#5624">:link:</a></sup></sub> fix
 
 * Fixed a bug where fly would no longer tell you if the team you logged in with was invalid
+
+#### <sub><sup><a name="5192" href="#5192">:link:</a></sup></sub> fix
+
+* @evanchaoli changed the behaviour of the web to retry individual build steps that fail when a worker disappears.
+
+#### <sub><sup><a name="5549" href="#5549">:link:</a></sup></sub> fix
+
+* Added a new Wrappa that returns HTTP 409 for endpoints listed in concourse/rfc#33 when the requested pipeine is archived
```

Comment:
* If the `wrappa` part of the codebase is a wrapper for Rata, consider just calling it a 'wrapper' here so that people unfamiliar with the joke will get what we're talking about in this context. 😄